cmd/util/flags/configflags/network.go

```diff
@@ -4,10 +4,12 @@ import "github.com/bacalhau-project/bacalhau/pkg/config/types"
 
 var NetworkFlags = []Definition{
 	{
-		FlagName:     "network",
-		ConfigPath:   types.NodeNetworkType,
-		DefaultValue: Default.Node.Network.Type,
-		Description:  `Inter-node network layer type (e.g. nats, libp2p).`,
+		FlagName:          "network",
+		ConfigPath:        types.NodeNetworkType,
+		DefaultValue:      Default.Node.Network.Type,
+		Description:       `Inter-node network layer type (e.g. nats, libp2p).`,
+		Deprecated:        true,
+		DeprecatedMessage: "The libp2p transport will be deprecated in a future version in favor of NATS",
```

Comment:
It's weird that we are deprecating this flag - will we just remove it in the future? (since there is no "changing your network"?

Reply:
https://github.com/bacalhau-project/bacalhau/pull/3966#discussion_r1586968545 Yup will remove the flag when we remove libp2p